app/backend/services/report-service.js

```diff
@@ -0,0 +1,32 @@
+var { getPool } = require('../config/db.js')
+var sql = require('mssql');
+
+async function getReport(user) {
+  try {
+    const pool = await getPool();
+    const reportInfo = await pool.request()
+      .input('userid', sql.VarChar(255), user.sub)
+      .query("SELECT * FROM report \
+              WHERE report.userid=@userid;");
+    return reportInfo.recordsets[0];
+  } catch (error) {
+    console.log(error);
+    return;
+  }
+}
+
+async function postReport(user, report) {
+  try {
+    const pool = await getPool();
+    const reportInfo = await pool.request()
+      .input("userid", sql.VarChar(255), user.sub)
+      .input("pomodoros", sql.Int, report.pomodoros)
+      .query('INSERT INTO report (userId, pomodoros) OUTPUT inserted.id, inserted.userid, inserted.pomodoros VALUES (@userid, @pomodoros)');
+    return reportInfo.recordsets[0];
+  } catch (error) {
+    console.log(error);
```

Comment:
Not that important, but is there a reason you use console.log(error) in some places instead of console.error(error)?

Reply:
Didn't know that was a thing lol. Will change it now